src/containers/App.tsx

```diff
@@ -24,6 +25,16 @@ function App() {
       >
         <Switch>
           <Route exact path="/events" component={HomeLandingPage} />
+          <Route
+            exact
+            path="/events/:eventID/new"
+            component={(props) => <CCPFormPage mode="new" {...props} />}
+          />
+          <Route
+            exact
+            path="/events/:eventID/ccps/:ccpID"
```

Comment:
the route you have right now is used for the ccp dashboard, my bad for not noticing until now
```suggestion
            path="/events/:eventID/ccps/:ccpID/edit"
```